### PR TITLE
Add startup triggers as a utility for other jobs

### PR DESCRIPTION
**Pull request by yunghuffy**
This reusable startup trigger should fold in with any DSL job except for pipelines easily.

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -426,3 +426,18 @@ new BaseJobBuilder(
     CommonUtils.addPerformancePublisher(delegate,failedThresholdPositive=10, failedThresholdNegative=10, unstableThresholdPositive=5, unstableThresholdNegative=5)
 }
 ```
+
+### Using a startup trigger
+
+```groovy
+mport jenkins.automation.builders.BaseJobBuilder
```
Comment (acrewdson) [inline docs/examples.md]: `import`

**Review by acrewdson — CHANGES_REQUESTED** [review]
one small change, otherwise looks good to me

File: docs/examples.md (hunk 1 above)
Comment (yunghuffy) [reply docs/examples.md]: Good catch

**Review by OrlandoSoto — APPROVED** [review]
👍

**Comment by yunghuffy** [comment]
@acrewdson changes submited

**Review by acrewdson — APPROVED** [review]

**Comment by yunghuffy** [comment]
Merge when ready on this one. Gonna give a little time for either @marcesher or @imuchnik to look at this.

**Review by imuchnik — APPROVED** [review]
LGTM

**Comment by acrewdson** [comment]
@champain you want to merge when ready?

**Pull request merged**